tests/functional/adapter/materialization/test_materialized_view.py

```diff
@@ -40,3 +47,97 @@ def test_mv_is_dropped_when_model_runs_view(self, project):
             "my_table": "table",
         }
         check_relation_types(project.adapter, expected)
+
+
+@pytest.mark.iceberg
+class TestIcebergMaterializedViewCreate:
+    # Configuration in dbt_project.yml
+    @pytest.fixture(scope="class")
+    def project_config_update(self):
+        return {
+            "name": "mv_test",
+            "models": {
+                "+materialized": "materialized_view",
+            },
+            "seeds": {
+                "+column_types": {"some_date": "timestamp(6)"},
+            },
+        }
+
+    # Everything that goes in the "seeds" directory
+    @pytest.fixture(scope="class")
+    def seeds(self):
+        return {
+            "seed.csv": seed_csv,
+        }
+
+    # Everything that goes in the "models" directory
+    @pytest.fixture(scope="class")
+    def models(self):
+        return {
+            "mat_view.sql": model_sql,
+        }
+
+    def test_mv_is_created_and_refreshed(self, project):
+        catalog = project.adapter.config.credentials.database
+        schema = project.adapter.config.credentials.schema
+
+        # Seed seed
+        results = run_dbt(["seed"], expect_pass=True)
+        assert len(results) == 1
+
+        # Create MV
+        results = run_dbt(["run"], expect_pass=True)
+        assert len(results) == 1
+
+        # Check if the data was loaded correctly
+        check_relations_equal(project.adapter, ["seed", "mat_view"])
+
+        # Add one row to seed
+        sql = f"""INSERT INTO {catalog}.{schema}.seed
+        VALUES (5, 'Mateo', timestamp '2014-09-07 17:04:27')""".strip()
```

Comment:
I don't think strip removes whitspace on every line but only at outer left and right.

```suggestion
        sql = f"""INSERT INTO {catalog}.{schema}.seed
        VALUES (5, 'Mateo', timestamp '2014-09-07 17:04:27')"""
```